src/workflows/steps.js

```diff
@@ -468,3 +470,152 @@ export function setPromote( state ) {
 export function verifyConflictResolution() {
 	return git.checkConflictMarkers();
 }
+
+export function getPackageScope( state ) {
+	if ( state.qa && typeof state.qa === "boolean" ) {
+		state.qa = "@lk";
+	} else if ( state.qa && state.qa.charAt( 0 ) !== "@" ) {
+		state.qa = `@${ state.qa }`;
+	}
+
+	return Promise.resolve();
+}
+
+export function getScopedRepos( state ) {
+	const { configPath, qa: scope } = state;
+	let content = {};
+	try {
+		content = util.readJSONFile( configPath );
+
+		let repos = Object.keys( content.devDependencies ).filter( key => key.includes( scope ) );
+		repos = repos.concat( Object.keys( content.dependencies ).filter( key => key.includes( scope ) ) );
+		repos = repos.map( key => key.slice( ( scope.length + 1 ), key.length ) );
```

Comment:
feels like this function can be refactored a bit with the same results. let me think about it for a bit